docs/developer/checkout/address.mdx

```diff
@@ -229,6 +229,14 @@ As a result, we get an updated checkout object with a delivery method set:
 }
 ```
 
+### Switching from C&C to standard delivery method
+
+After choosing the click & collect option on a checkout, going back to a standard delivery method will require the shipping address to be set, this can't be done directly by checkoutDeliveryMethodUpdate, but can be achieved using the flow below.
```

Comment:
This sounds for me a little bit like we have a bug in our code. IMO we should describe it as an expected flow, and maybe add a information why the shipping address cannot be just changed. 
- `going back to a standard delivery method will require the shipping address to be set` - the checkout already have the shipping address set but it points to CC, so it need to be changed not set
- `going back to a standard delivery method will require the shipping address to be set, this can't be done directly by checkoutDeliveryMethodUpdate` - but the address is not updated by `checkoutDeliveryMethodUpdate` mutation, so this sentence is misleading